web/renderer/components/ConnectionsAndDatabases/ConnectionItem.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  DatabaseConnection,
+  DatabaseConnectionFragment,
+} from "@gen/graphql-types";
+import { MdRemoveRedEye } from "@react-icons/all-files/md/MdRemoveRedEye";
+import { FaChevronRight } from "@react-icons/all-files/fa/FaChevronRight";
+import { Button } from "@dolthub/react-components";
+import { excerpt } from "@dolthub/web-utils";
+import cx from "classnames";
+import { StateType } from "./useSelectedConnection";
+import { DatabaseTypeLabel } from "./DatabaseTypeLabel";
+import css from "./index.module.css";
+
+type Props = {
+  conn: DatabaseConnectionFragment;
+  state: StateType;
```

Comment:
Do you need to pass the whole `state`? Seems like you only need `connection` (which could use a clearer name to distinguish from `conn`)?